sources/Interop/Windows/um/winnt/Windows.Manual.cs

```diff
@@ -3,199 +3,5421 @@
+
+        public const int X3_IMM20_INST_WORD_X = 3;
+
+        public const int X3_IMM20_SIZE_X = 20;
+
+        public const int X3_IMM20_INST_WORD_POS_X = 4;
+
+        public const int X3_IMM20_SIGN_VAL_POS_X = 0;
+
+        public const int X3_IMM39_1_INST_WORD_X = 2;
+
+        public const int X3_IMM39_1_SIZE_X = 23;
+
+        public const int X3_IMM39_1_INST_WORD_POS_X = 0;
+
+        public const int X3_IMM39_1_SIGN_VAL_POS_X = 36;
+
+        public const int X3_IMM39_2_INST_WORD_X = 1;
+
+        public const int X3_IMM39_2_SIZE_X = 16;
+
+        public const int X3_IMM39_2_INST_WORD_POS_X = 16;
+
+        public const int X3_IMM39_2_SIGN_VAL_POS_X = 20;
+
+        public const int X3_P_INST_WORD_X = 3;
+
+        public const int X3_P_SIZE_X = 4;
+
+        public const int X3_P_INST_WORD_POS_X = 0;
+
+        public const int X3_P_SIGN_VAL_POS_X = 0;
+
+        public const int X3_TMPLT_INST_WORD_X = 0;
+
+        public const int X3_TMPLT_SIZE_X = 4;
+
+        public const int X3_TMPLT_INST_WORD_POS_X = 0;
+
+        public const int X3_TMPLT_SIGN_VAL_POS_X = 0;
+
+        public const int X3_BTYPE_QP_INST_WORD_X = 2;
+
+        public const int X3_BTYPE_QP_SIZE_X = 9;
+
+        public const int X3_BTYPE_QP_INST_WORD_POS_X = 23;
+
+        public const int X3_BTYPE_QP_INST_VAL_POS_X = 0;
+
+        public const int X3_EMPTY_INST_WORD_X = 1;
+
+        public const int X3_EMPTY_SIZE_X = 2;
+
+        public const int X3_EMPTY_INST_WORD_POS_X = 14;
+
+        public const int X3_EMPTY_INST_VAL_POS_X = 0;
+
+        public const int IMAGE_REL_BASED_ABSOLUTE = 0;
+
+        public const int IMAGE_REL_BASED_HIGH = 1;
+
+        public const int IMAGE_REL_BASED_LOW = 2;
+
+        public const int IMAGE_REL_BASED_HIGHLOW = 3;
+
+        public const int IMAGE_REL_BASED_HIGHADJ = 4;
+
+        public const int IMAGE_REL_BASED_MACHINE_SPECIFIC_5 = 5;
+
+        public const int IMAGE_REL_BASED_RESERVED = 6;
+
+        public const int IMAGE_REL_BASED_MACHINE_SPECIFIC_7 = 7;
+
+        public const int IMAGE_REL_BASED_MACHINE_SPECIFIC_8 = 8;
+
+        public const int IMAGE_REL_BASED_MACHINE_SPECIFIC_9 = 9;
+
+        public const int IMAGE_REL_BASED_DIR64 = 10;
+
+        public const int IMAGE_REL_BASED_IA64_IMM64 = 9;
+
+        public const int IMAGE_REL_BASED_MIPS_JMPADDR = 5;
+
+        public const int IMAGE_REL_BASED_MIPS_JMPADDR16 = 9;
+
+        public const int IMAGE_REL_BASED_ARM_MOV32 = 5;
+
+        public const int IMAGE_REL_BASED_THUMB_MOV32 = 7;
+
+        public const int IMAGE_ARCHIVE_START_SIZE = 8;
+
+        // !<arch>
+        public static ReadOnlySpan<sbyte> IMAGE_ARCHIVE_START => new sbyte[] { 0x21, 0x3C, 0x61, 0x72, 0x63, 0x68, 0x3E, 0x0A, 0x00 };
+
+        // `
+        public static ReadOnlySpan<sbyte> IMAGE_ARCHIVE_END => new sbyte[] { 0x60, 0x0A, 0x00 };
+
+        // 
+        public static ReadOnlySpan<sbyte> IMAGE_ARCHIVE_PAD => new sbyte[] { 0x0A, 0x00 };
+
+        // /               
```

Comment:
Nothing to do here, but these strings aren't super visible 😄 
15 trailing spaces...

Reply:
Yeah, that threw me at first. I thought some comments were being left in by either ClangSharp or my script, and it was my script... as expected, given the byte arrays are all spaces and newlines and stuff.

Reply:
Maybe quotes could be added and the well known characters could be inserted as escapes (`\n`, `\r`, `\t`, and `\0`)?
Not particular important, but it might help